src/replicache.ts

```diff
@@ -447,6 +466,42 @@ export class Replicache<MD extends MutatorDefs = {}> {
     resolveLicenseCheck(true);
   }
 
+  private async _startLicenseActive(
+    resolveLicenseActive: (valid: boolean) => void,
+    lc: LogContext,
+  ): Promise<() => void> {
+    if (
+      this._licenseKey === undefined ||
+      this._licenseKey === TEST_LICENSE_KEY
+    ) {
+      resolveLicenseActive(false);
+      return noop;
+    }
+
+    const activate = async () => {
```

Comment:
The use of the word "active" here seems to be overloading the definition of "active" as a license that requires pings. Why not call this function `pingLicense` or something?

Reply:
Ah yes, ping is still used to eep the socket alive in Reflect.

OK I just had a hard time parsing this. The word "activate" typically means something you do once, not recurring. Maybe *markActive*?

Reply:
I might even say `markLicenseActive`. I usually prefer your style of defining names up front to encapsulate concepts and decrease verbosity, but in this case I just can't think of a good one :).

Reply:
yeah neither could i. markactive seems workable. any chance you want to call the thingy that keeps sockets alive 'marco polo' or 'keepopen ack' or 'yo supdog' or something?

i will do markactive once i get an answer on whether i can use mixins